<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesetEditor</class>
 <widget class="QWidget" name="TilesetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PoryPal - Tileset Editor</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="toolbar_layout">
     <item>
      <widget class="QPushButton" name="btn_load_tileset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>üß© Load Tileset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_tileset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>üíæ Save Tileset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_load_preset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>üíΩ Load Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_preset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>üíø Save Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_help">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>‚ùî Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInput">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="input_container" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="input_layout">
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_input">
          <property name="text">
           <string>Input Tileset</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="input_scroll_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>958</width>
             <height>276</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGraphicsView" name="input_view">
              <property name="dragMode">
               <enum>QGraphicsView::RubberBandDrag</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="input_settings_layout">
          <item>
           <widget class="QLabel" name="label_tile_width">
            <property name="text">
             <string>Tile Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_tile_width">
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_tile_height">
            <property name="text">
             <string>Tile Height:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_tile_height">
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_apply_grid">
            <property name="text">
             <string>Apply Grid</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_info_label_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="output_container" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="output_layout">
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_output">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Output Tileset</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="output_scroll_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>946</width>
             <height>276</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGraphicsView" name="output_view">
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="output_settings_layout">
          <item>
           <widget class="QLabel" name="label_columns">
            <property name="text">
             <string>Columns:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_columns">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_rows">
            <property name="text">
             <string>Rows:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_rows">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_scale">
            <property name="text">
             <string>Scale:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_scale">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_create_layout">
            <property name="text">
             <string>Create Layout</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear_layout">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_info_label_output">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
